src/promptflow/promptflow/_telemetry/mdc_exporter.py

```diff
@@ -0,0 +1,50 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import json
+from typing import Sequence
+
+from opentelemetry.sdk.trace import ReadableSpan
+from opentelemetry.sdk.trace.export import SpanExporter
+
+
+class MdcExporter(SpanExporter):
+    """An open telemetry span exporter to MDC."""
```

Comment:
what's the full name for MDC?

Reply:
MDC = Machine Learning Data Collection
[azureml-ai-monitoring - Repos (visualstudio.com)](https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2?version=GBmain&path=/src/azureml-ai-monitoring)